File: euser-sso/src/main/java/com/example/eusersso/config/AfRedisProperties.java
package com.example.eusersso.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties(
        prefix = "spring.data.af-redis"
)
@Data
public class AfRedisProperties {

    private int database;

    private String host;

    private int port;
}

File: api-factory/af-gateway/src/main/kotlin/com/example/afgateway/config/AfRedisProperties.kt
package com.example.afgateway.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties(prefix = "spring.data.af-redis")
data class AfRedisProperties(
    val database: Int = 0,
    val host: String = "localhost",
    val port: Int = 6379,
)
